src/cisco-decrypt.c

```diff
@@ -1,21 +1,12 @@
-/* Decoder for password encoding of Cisco VPN client.
-   Copyright (C) 2005 Maurice Massar
+/*
+  Decoder for password encoding of Cisco VPN client.
 
-   Thanks to HAL-9000@evilscientists.de for decoding and posting the algorithm!
+  SPDX-FileCopyrightText: 2023 Jolla Ltd.
+  SPDX-FileCopyrightText: 2005 Maurice Massar
 
-   This program is free software; you can redistribute it and/or modify
-   it under the terms of the GNU General Public License as published by
-   the Free Software Foundation; either version 2 of the License, or
-   (at your option) any later version.
+  SPDX-License-Identifier: GPL-2.0-or-later
 
-   This program is distributed in the hope that it will be useful,
-   but WITHOUT ANY WARRANTY; without even the implied warranty of
-   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-   GNU General Public License for more details.
-
-   You should have received a copy of the GNU General Public License
-   along with this program; if not, write to the Free Software
-   Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+  Thanks to HAL-9000@evilscientists.de for decoding and posting the algorithm!
```

Comment:
Please move this right after the header `Decoder for password encoding of Cisco VPN client.`.